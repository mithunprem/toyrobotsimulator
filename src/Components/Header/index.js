import React, { Fragment } from 'react';

const Header = () => {
  return (
    <Fragment>
      <header className="toy-robot-simulator-header">
        <h5 className="ml-3 header-text">Toy Robot Simulator</h5>
      </header>
    </Fragment>
  );
}

export default Header;
