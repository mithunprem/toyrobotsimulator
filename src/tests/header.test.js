import React from 'react';
import { create } from 'react-test-renderer';
import Header from '../components/header';

describe("Header", () => {
  test("should render without throwing an error", () => {
    const header = create(<Header />).toJSON()
    expect(header).toMatchSnapshot()
  });
});
