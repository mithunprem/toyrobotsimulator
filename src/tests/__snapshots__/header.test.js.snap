// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render without throwing an error 1`] = `
<header
  className="toy-robot-simulator-header"
>
  <h5
    className="ml-3 header-text"
  >
    Toy Robot Simulator
  </h5>
</header>
`;
